nfqueue_gteq_1.12.go

```diff
@@ -38,7 +38,8 @@ func Open(config *Config) (*Nfqueue, error) {
 		return nil, ErrInvFlag
 	}
 
-	con, err := netlink.Dial(unix.NETLINK_NETFILTER, &netlink.Config{NetNS: config.NetNS})
+	// Disable the network namespace goroutine if no explicit network namespace was specified
```

Comment:
Comment is slightly off, maybe: 
// Disable netlink goroutine spawned in a  different namespce, if no explicit network namespace is specified in the config